sky/backends/cloud_vm_ray_backend.py

```diff
@@ -1996,6 +1997,22 @@ def provision_with_retries(
                                        skip_unnecessary_provisioning else None)
 
         failover_history: List[Exception] = list()
+        # When jobs controller/server using the local credentials which are
+        # expiring it may cause the cluster to be leaked. So, checking the
+        # enabled clouds and expiring credentials and warning the user to use
+        # the credentials that never expire or a service account.
+        if task.is_controller_task():
+            enabled_clouds = sky_check.get_cached_enabled_clouds_or_refresh()
+            expirable_clouds = backend_utils.get_expirable_clouds(
+                enabled_clouds)
+
+            if len(expirable_clouds) > 0:
+                warnings = (f'\033[93mWarning: Credentials used for '
+                            f'{expirable_clouds} may expire.Clusters may be '
+                            f'leaked if the credentials expire while jobs '
+                            f'are running.It is recommended to use credentials'
+                            f' that never expire or a service account.\033[0m')
```

Comment:
Need to add spaces
<img width="752" alt="image" src="https://github.com/user-attachments/assets/9a8a7c60-0eb9-4885-904f-0416689efacf" />

```suggestion
                warnings = (f'\033[93mWarning: Credentials used for '
                            f'{expirable_clouds} may expire. Clusters may be '
                            f'leaked if the credentials expire while jobs '
                            f'are running. It is recommended to use credentials'
                            f' that never expire or a service account.\033[0m')
```